Makefile

```
@@ -42,6 +46,46 @@ bindings:
 test:
 	go test ./...
 
+
+get-delegation-manager-address:
+	@sed -n 's/.*"delegationManager": "\([^"]*\)".*/\1/p' contracts/script/output/devnet/eigenlayer_deployment_output.json
+
+operator-generate-keys:
+	@echo "Generating BLS keys"
+	eigenlayer operator keys create --key-type bls --insecure operator
+	@echo "Generating ECDSA keys"
+	eigenlayer operator keys create --key-type ecdsa --insecure operator
+
+operator-generate-config:
+	@echo "Generating operator config"
+	eigenlayer operator config create
+
+operator-get-eth:
+	@echo "Sending funds to operator address on devnet"
+	@. ./scripts/fund_operator_devnet.sh
+
+operator-register-with-eigen-layer:
+	@echo "Registering operator with EigenLayer"
+	@eigenlayer operator register operator/config/devnet/operator.yaml
```

```suggestion
	@echo "" | eigenlayer operator register operator/config/devnet/operator.yaml
```
this sets the the passphrase to an empty string so that we don't need user input